castero/players/vlcplayer.py

```diff
@@ -88,9 +89,18 @@ def seek(self, direction, amount) -> None:
         assert direction == 1 or direction == -1
         if self._player is not None:
             self._player.set_time(
-                self._player.get_time() + (direction * amount * 1000)
+                self._player.get_time() + (direction * amount *
+                    constants.MILLISECONDS_IN_SECOND)
             )
 
+    def seek_from_start(self, amount) -> None:
+        """Seek forward amount from start.
+
+        Overrides method from Player; see documentation in that class.
+        """
+        if self._player is not None:
+            self._player.set_time((amount * constants.MILLISECONDS_IN_SECOND))
```

Comment:
This crashes currently because set_time only takes ints. Just adding a cast here made this work perfectly, though!